test/functions/whereClauses/whereClauseParamArgMismatch3.good

```diff
@@ -1,2 +1,2 @@
 whereClauseParamArgMismatch3.chpl:10: error: unresolved call 'foo(1.2)'
-whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(8))
+whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(64))
```

Comment:
Always good to get evidence that I haven't secretly given up my straight-edge lifestyle! (particularly at work!)